src/borg/archiver.py

```diff
@@ -3776,6 +3776,11 @@ def define_borg_mount(parser):
                                help='Sort the output lines by file path.')
         subparser.add_argument('--json-lines', action='store_true',
                                help='Format output as JSON Lines. ')
+        subparser.add_argument('--format', '--diff-format', metavar='FORMAT', dest='format',
```

Comment:
iirc we had `--list-format` for borg list, but just deprecated it / got rid of it, only using `--format` in future.

so, guess we also only want `--format` here.

Reply:
Ok, fine. Thank you for the comment.